### PR TITLE
Started README.md

### PR DESCRIPTION
**Pull request by charlietaoxl**
- Added Goal
- Added MVP requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+Goal: Modify a screenshot to filter out colours in a way that differentiates the colours better for protanopia.
+
+
+
+MVP
+1. Continuously check the screenshot folder for a new screenshot
+2. Apply some colour changes onto the screenshot
+3. Display the new image 
+4. Be able to close the displayed image
+
```
Comment (rantaoca) [inline README.md]: Unnecessary trailing whitespace, please remove

**Review by rantaoca — CHANGES_REQUESTED** [review]
Good description. Please fix the requested changes

File: README.md (hunk 1 above)
Comment (charlietaoxl) [reply README.md]: Thank you so much! I will get to that right away~ :D T.T

Comment (charlietaoxl) [reply README.md]: I have made the changes.

**Review by rantaoca — APPROVED** [review]
Looks good to me

**Pull request merged**